src/api/request-hooks/hook.js

```diff
@@ -17,12 +17,13 @@ export default class RequestHook {
         return [RequestFilterRule.ANY];
     }
 
-    _instantiateRequestFilterRules () {
+    instantiateRequestFilterRules () {
```

Comment:
let's leave it "private": `_instantiateRequestFilterRules`

Reply:
But the behavior didn't change.
Plus, I think better to leave private methods with private indicator `_` in public API class.

Reply:
Ok. Let it be.